android/build.gradle

```diff
@@ -1,23 +1,26 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
     ext {
-        buildToolsVersion = "33.0.0"
+        buildToolsVersion = "34.0.0"
         minSdkVersion = 21
-        compileSdkVersion = 33
-        targetSdkVersion = 33
+        compileSdkVersion = 34
+        targetSdkVersion = 34
         // needed by camera module
         googlePlayServicesVersion = "17+"
         supportLibVersion = "28.0.0"
         lifecycleVersion = "2.0.0"
-        kotlin_version = "1.8.21"
+        kotlin_version = "1.9.0"
+        kotlin_version = "1.9.0"
+        compose_version = "1.5.2"
+        nav_version = "2.7.7"
         androidx_core_version = "1.6.0"
         androidXBrowser = "1.2.0"
         excludeAppGlideModule = true
         androidx_lifecycle_version = "2.6.1"
         constraint_layout_version = "2.0.4"
         appcompat_version = "1.3.1"
         excludeAppGlideModule = true
-        compose_ui_version = '1.2.0'
+        compose_ui_version = '1.5.4'
         camerax_version = "1.3.0-alpha04"
 
         // We use NDK 23 which has both M1 support and is the side-by-side NDK version from AGP.
```

Comment:
Don't we need to upgrade the NDK version from `23.1.7779620` to `25.1.8937393`?

Reply:
> Don't we need to upgrade the NDK version from `23.1.7779620` to `25.1.8937393`?

yep we need. I've updated, thanks